### PR TITLE
add wdio tests to TFE

### PR DESCRIPTION
**Pull request by Ditmir-Vega**

### CODE REVIEW

File: automation/features/pageobjects/vesting.page.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import Page from "./page";
+
+class VestingPage extends Page {
+  get trancheItemTestId() {
+    return "tranche-item";
+  }
+  get validatorNodeTitle() {
+    return $('data-test-id="validator-node-title"]');
```
Comment (mattrussell36) [inline automation/features/pageobjects/vesting.page.js:8]: Missing a square bracket here. Probably worth checking any tests here to make sure they aren't evergreen

File: automation-cypress-old/.gitignore [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+*.swp
+cypress/videos
+
```
Comment (mattrussell36) [inline automation-cypress-old/.gitignore:4]: Do we need to keep `automation-cypress-ld/` around. Would prefer to just delete it if possible

File: automation/commands.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    getByTestId: function (testId) {
+        return $(`[data-testid="${testId}"]`)
+    }
+
```
Comment (mattrussell36) [inline automation/commands.js:5]: formatting here

**Review by mattrussell36 — COMMENTED** [review]
Had a quick look over this, left a few comments, worth running prettier over this to keep the formatting nice and clean.

File: automation/features/pageobjects/vesting.page.js (hunk 1 above)
Comment (Ditmir-Vega) [reply automation/features/pageobjects/vesting.page.js:8]: veryyyyyyy observant thanks !

File: automation-cypress-old/.gitignore (hunk 2 above)
Comment (Ditmir-Vega) [reply automation-cypress-old/.gitignore:4]: will move it to a seperate repo somewhere

**Pull request closed without merging**